central/complianceoperator/v2/checkresults/datastore/datastore.go

```diff
@@ -43,3 +49,11 @@ type DataStore interface {
 func New(store store.Store, db postgres.DB, searcher search.Searcher) DataStore {
 	return &datastoreImpl{store: store, db: db, searcher: searcher}
 }
+
+// GetTestPostgresDataStore provides a datastore connected to postgres for testing purposes.
+func GetTestPostgresDataStore(_ *testing.T, pool postgres.DB) (DataStore, error) {
+	store := store.New(pool)
+	indexer := checkResultsStorage.NewIndexer(pool)
+	searcher := checkresultsSearch.New(store, indexer)
+	return New(store, pool, searcher), nil
+}
```

Comment:
```suggestion
func GetTestPostgresDataStore(_ *testing.T, pool postgres.DB) DataStore {
	store := store.New(pool)
	indexer := checkResultsStorage.NewIndexer(pool)
	searcher := checkresultsSearch.New(store, indexer)
	return New(store, pool, searcher)
}
```

Reply:
Nothing in there generates an error, so we shouldn't return one.  I have been slowly fixing these as I encounter them.